Create good-citizen.md (#85)
exercism/docs: good-citizen.md

# Being a Good Citizen of Exercism 

We want Exercism to be a great place to spend time, whoever you are.
To ensure that Exercism is that, we place a strong focus on being good community citizens.
This is formalised in our [Code of Conduct](https://exercism.io/code-of-conduct) but can be summed up simply in one sentence: **Be kind.**

## The words we use

There are lots of ways to embrace being kind within Exercism but the most powerful is to consider how your words affect others.

Words in themselves are meaningless. 
They are vibrations caused by puffs of air coming out of our lungs.
However, because of **context**, words are powerful.

It's easy to know how you intend your words to come across, but it's difficult to know how others hear your words.
What we say is strongly tempered by our experience in life, our cultural norms, and our primary spoken language.
We believe it's better to choose words that are **clear** and **unambiguous** across cultures, and where possible, across spoken languages.

> This means being careful when using metaphors, acroynyms or phrases that might only be understandable to someone from a specific background or with specific knowledge.

Author: ```suggestion
This means being careful when using metaphors, acronyms or phrases that might only be understandable to someone from a specific background or with specific knowledge.
It also means being open to learning about how people from different cultures might misunderstand your intentions, and changing the words you use accordingly.
```

One example is that within Exercism we ask people not to use the word "guys" unless explicitely addressing a group of men. 
While in many places the word "guys" can refer to a group of people irrespective of gender, in many places it does explicltely mean "men".

> So, regardless of the intention of the speaker, the person hearing the word "guys" understands the speaker to be only addressing only men, and feels excluded.

Author: ```suggestion
So, regardless of the intention of the speaker, people from those cultures hearing the word "guys" understands the speaker to be only addressing only men, and if they are not male, feels excluded.
```

Reviewer: I can't leave a suggestion to a suggestion..so putting it here:

So regardless of the intention of the speaker,  people from languages and cultures that use "guys" to refer only to men would interpret it as only _addressing_ men -- and if they are not male, they would feel excluded.

# Being a Good Citizen of Exercism 

We want Exercism to be a great place to spend time, whoever you are.
To ensure that Exercism is that, we place a strong focus on being good community citizens.
This is formalised in our [Code of Conduct](https://exercism.io/code-of-conduct) but can be summed up simply in one sentence: **Be kind.**

## The words we use

There are lots of ways to embrace being kind within Exercism but the most powerful is to consider how your words affect others.

Words in themselves are meaningless. 
They are vibrations caused by puffs of air coming out of our lungs.
However, because of **context**, words are powerful.

> It's easy to know how you intend your words to come across, but it's difficult to know how others hear your words.

Reviewer: ```suggestion
It's easy to know how you intend _your_ words to come across, but it's difficult to know how others _hear your words_.
```

Author: ```suggestion
It's easy to know how you intend your words to _come across_, but it's difficult to know how others _hear your words_.
```

Author: Thoughts on my version @BethanyG - I tried pairing the two phrases slightly differently. See which you think is clearer :)

Reviewer: I like the second one.  😄   Better emphasis -- and better example of the problem.

> What we say is strongly tempered by our experience in life, our cultural norms, and our primary spoken language.

Reviewer: ```suggestion
What we say is strongly tempered by our experiences in life, our cultural norms, and our primary spoken language.
```

We believe it's better to choose words that are **clear** and **unambiguous** across cultures, and where possible, across spoken languages.
This means being careful when using metaphors, acronyms or phrases that might only be understandable to someone from a specific background or with specific knowledge.
It also means being open to learning about how people from different cultures might misunderstand your intentions, and changing the words you use accordingly.

One example is that within Exercism we ask people not to use the word "guys" unless explicitely addressing a group of men. 

> While in many places the word "guys" can refer to a group of people irrespective of gender, in many places it does explicltely mean "men".

Reviewer: ```suggestion
While in many places the word "guys" can refer to a group of people irrespective of gender, in many other places it explicitly means "men".
```

So, regardless of the intention of the speaker, the person hearing the word "guys" understands the speaker to be only addressing only men, and feels excluded.

> In tech, where women are in the strong miniority, this adds yet another sense of negativity and exclusion.

Author: ```suggestion
In tech, where men are in the strong majority, this adds yet another sense of negativity and exclusion to everyone else.
```

Reviewer: In tech, where those who identify as men are in the strong majority, this adds yet another sense of negativity and exclusion to anyone who does not identify in that majority.

> For some people who are used to using the word "guys" to mean a group of people irrespective of gender, this rule can seem pedantic or irrational.

Author: ```suggestion
At first, many people find this rule to be pedantic or irrational, as they are clear in their intentions to not exclude anyone.
```

> However, when guided by the principle of using communication that is clear and unambiguous, and driven by being kind to others in the community, it becomes clear that the word "guys" does not convey the intended meaning to the receiver and may cause a sense of exclusion, and so we choose to use clearer words.

Author: ```suggestion
However, when guided by the principle of communicating in a way that is clear and unambiguous, and driven by the desire to be kind to others in the community, it becomes clear that the word "guys" does not necessarily convey the intended meaning to everyone, and so they choose to use clearer words.
```

~~~~note
Our co-founder, Jeremy, is from the UK, where the British dictionary explicitely states that "guys" is a gender neutral word.
In real life, he commonly addresses groups of women using the word "guys". 

> However, in an international setting such as conferences, or within on multi-national platform such as Exercism, he would choose to use a clearer word such as "folks" (or occasionally y'all when he's feeling particuarly Texan)

Reviewer: ```suggestion
However, in an international setting such as conferences, or within a multi-national platform such as Exercism, he would choose to use a clearer word such as "folks" (or occasionally y'all when he's feeling particularly Texan)
```

> ~~~~

Author: ```suggestion
~~~~

## Remember that we're all human

Just as it is important to say the right things, it is equally important to understand that people who cause offence might have had no intention to do so.
Cultural differences mean that people often say things that would be clear to them and their friends to mean one thing, but mean something entirely different to you.
In those situations, **give the person the benefit of the doubt.** 
It is always best to presume that someone is acting with the intentions of being clear and kind, and that they are unaware of how their words will affect others.
As with so much of Exercism, you don't know what you don't know, and it's easy to accidentally say the wrong thing.

The important thing is not the mistake, it is the learning and the change that comes from it.
So if someone says something that you feel is offensive, and you feel comfortable doing so, feel free to tell them.
And if someone tells you you've said something that is offensive to them, believe them, and embrace the experience of learning more about that other culture, and extending your communication skills to not cause offense in future.

The beauty of Exercism is about making mistakes and learning from them. 
Exercism is brilliant at helping with that through mentoring, but it's also brilliant at introducing you to new ideas and information through our huge multinational community.
Embrace it!

```

# Being a Good Citizen of Exercism 

We want Exercism to be a great place to spend time, whoever you are.
To ensure that Exercism is that, we place a strong focus on being good community citizens.
This is formalised in our [Code of Conduct](https://exercism.io/code-of-conduct) but can be summed up simply in one sentence: **Be kind.**

## The words we use

There are lots of ways to embrace being kind within Exercism but the most powerful is to consider how your words affect others.

Words in themselves are meaningless. 
They are vibrations caused by puffs of air coming out of our lungs.
However, because of **context**, words are powerful.

It's easy to know how you intend your words to come across, but it's difficult to know how others hear your words.
What we say is strongly tempered by our experiences in life, our cultural norms, and our primary spoken language.

> We believe it's better to choose words that are **clear** and **unambiguous** across cultures, and where possible, across spoken languages.

Author: ```suggestion
At Exercism, believe it's better to choose words that are **clear** and **unambiguous** across cultures, and where possible, across spoken languages.
```

> At Exercism, believe it's better to choose words that are **clear** and **unambiguous** across cultures, and where possible, across spoken languages.

Author: ```suggestion
At Exercism, believe it's better to choose words that are **clear** and **unambiguous** across cultures and, where possible, across spoken languages.
```

This means being careful when using metaphors, acronyms or phrases that might only be understandable to someone from a specific background or with specific knowledge.
It also means being open to learning about how people from different cultures might misunderstand your intentions, and changing the words you use accordingly.

One example is that within Exercism we ask people not to use the word "guys" unless explicitely addressing a group of men. 
While in many places the word "guys" can refer to a group of people irrespective of gender, in many other places it explicitly means "men".

> So, regardless of the intention of the speaker, people from those cultures hearing the word "guys" understands the speaker to be only addressing only men, and if they are not male, feels excluded.

Author: ```suggestion
So, regardless of the intention of the speaker, a person from that cultures hearing the word "guys" understands the speaker to be only addressing only men and, if they are not male, feels excluded.
```

In tech, where men are in the strong majority, this adds yet another sense of negativity and exclusion to everyone else.

At first, many people find this rule to be pedantic or irrational, as they are clear in their intentions to not exclude anyone.
However, when guided by the principle of communicating in a way that is clear and unambiguous, and driven by the desire to be kind to others in the community, it becomes clear that the word "guys" does not necessarily convey the intended meaning to everyone, and so they choose to use clearer words.

~~~~note
Our co-founder, Jeremy, is from the UK, where the British dictionary explicitely states that "guys" is a gender neutral word.
In real life, he commonly addresses groups of women using the word "guys". 

> However, in an international setting such as conferences, or within on multi-national platform such as Exercism, he would choose to use a clearer word such as "folks" (or occasionally y'all when he's feeling particuarly Texan)

Author: ```suggestion
However, in an international setting such as conferences, or within on multi-national platform such as Exercism, he would choose to use a clearer word such as "folks" (or occasionally y'all when he's feeling particularly Texan).
As such, there is no judgement about whether the word "guys" should mean one thing or another or is good or bad, purely that the usage of the word should change with context.
```

~~~~

> ## Remember that we're all human

Author: ```suggestion
## You don't know what you don't know
```

Just as it is important to say the right things, it is equally important to understand that people who cause offence might have had no intention to do so.
Cultural differences mean that people often say things that would be clear to them and their friends to mean one thing, but mean something entirely different to you.
In those situations, **give the person the benefit of the doubt.** 
It is always best to presume that someone is acting with the intentions of being clear and kind, and that they are unaware of how their words will affect others.

> As with so much of Exercism, you don't know what you don't know, and it's easy to accidentally say the wrong thing.

Author: ```suggestion
It's easy to not know something will cause offence and accidentally say the wrong thing.
```

# Being a Good Citizen of Exercism 

We want Exercism to be a great place to spend time, whoever you are.

> To ensure that Exercism is that, we place a strong focus on being good community citizens.

Reviewer 2: ```suggestion
To ensure that Exercism is that, we place a strong focus on being good community members.
```

This is formalised in our [Code of Conduct](https://exercism.io/code-of-conduct) but can be summed up simply in one sentence: **Be kind.**

## The words we use

> There are lots of ways to embrace being kind within Exercism but the most powerful is to consider how your words affect others.

Reviewer 3: ```suggestion
There are lots of ways to embrace being kind within Exercism, but the most powerful is to consider how your words affect others.
```

Words in themselves are meaningless. 

> They are vibrations caused by puffs of air coming out of our lungs.

Reviewer 2: ```suggestion
They are vibrations caused by puffs of air coming out of our lungs.
```

This doesn't apply to Exercism, where almost all communication happens through writing.

However, because of **context**, words are powerful.

> It's easy to know how you intend your words to come across, but it's difficult to know how others hear your words.

Reviewer 2: Same here, perhaps "understand your words" or "perceive your words" is more fitting than "hear" due to the text-based communication.

> What we say is strongly tempered by our experiences in life, our cultural norms, and our primary spoken language.

Reviewer 4: Can we use "native language" instead of spoken language? For people in the Deaf community their native language isn't spoken.

> At Exercism, believe it's better to choose words that are **clear** and **unambiguous** across cultures and, where possible, across spoken languages.

Reviewer: ```suggestion
At Exercism, we believe it's better to choose words that are **clear** and **unambiguous** across cultures and, where possible, across spoken languages.
```

> At Exercism, we believe it's better to choose words that are **clear** and **unambiguous** across cultures and, where possible, across spoken languages.

Reviewer 4: I think in this context "spoken" isn't referring to native language but rather to human languages as opposed to programming languages. Maybe we can use "natural languages" instead?

This means being careful when using metaphors, acronyms or phrases that might only be understandable to someone from a specific background or with specific knowledge.
It also means being open to learning about how people from different cultures might misunderstand your intentions, and changing the words you use accordingly.

> One example is that within Exercism we ask people not to use the word "guys" unless explicitely addressing a group of men. 

Reviewer 2: As I said on Slack, I'd argue it's a bad word to use even if you address a group of men for the exact same reasons you outline below. It's more clear to use a word that's gendered everywhere, and not just in certain variations of English.

While in many places the word "guys" can refer to a group of people irrespective of gender, in many other places it explicitly means "men".

> So, regardless of the intention of the speaker, a person from that cultures hearing the word "guys" understands the speaker to be only addressing only men and, if they are not male, feels excluded.

Reviewer: ```suggestion
So, regardless of the intention of the speaker, a person from that cultural background hearing the word "guys" understands the speaker to be only addressing men and, if they are not male, feels excluded.
```

> So, regardless of the intention of the speaker, a person from that cultural background hearing the word "guys" understands the speaker to be only addressing men and, if they are not male, feels excluded.

Reviewer 2: ```suggestion
So, regardless of the intention of the speaker, a person from that cultural background hearing the word "guys" understands the speaker to be only addressing men and, if they are not male, may feel excluded.
```

Author: I'm going to leave this one because if someone is non-male and hearing someone address only males, I think they will always feel excluded. (If they know the person intends to speak in a genderless way, then they are probably choosing to be offended/excluded, in which case I'd make the change to the sentence)

In tech, where men are in the strong majority, this adds yet another sense of negativity and exclusion to everyone else.

At first, many people find this rule to be pedantic or irrational, as they are clear in their intentions to not exclude anyone.
However, when guided by the principle of communicating in a way that is clear and unambiguous, and driven by the desire to be kind to others in the community, it becomes clear that the word "guys" does not necessarily convey the intended meaning to everyone, and so they choose to use clearer words.

~~~~note
Our co-founder, Jeremy, is from the UK, where the British dictionary explicitely states that "guys" is a gender neutral word.
In real life, he commonly addresses groups of women using the word "guys". 

> However, in an international setting such as conferences, or within on multi-national platform such as Exercism, he would choose to use a clearer word such as "folks" (or occasionally y'all when he's feeling particularly Texan).

Author: ```suggestion
However, in an international setting such as conferences, or within a multi-national platform such as Exercism, he would choose to use a clearer word such as "folks" (or occasionally y'all when he's feeling particularly Texan).
```

As such, there is no judgement about whether the word "guys" should mean one thing or another or is good or bad, purely that the usage of the word should change with context.
~~~~

> ## You don't know what you don't know

Author: ```suggestion
### You don't know what you don't know
```

Just as it is important to say the right things, it is equally important to understand that people who cause offence might have had no intention to do so.

> Cultural differences mean that people often say things that would be clear to them and their friends to mean one thing, but mean something entirely different to you.

Reviewer 2: Perhaps it's also worth mentioning language barriers. Some people may simply lack the vocabulary, eloquence or language skills to both be aware of the potentially hurtful meaning and, even if they are aware, to know how to phrase it differently to avoid it. For example, I've had ten years of English classes in school, yet the singular they wasn't mentioned a single time and it has no equivalent in German, so if I didn't learn about it "by accident" through picking it up from others, I'd not even be aware that it existed and would just use "he" by default.

In those situations, **give the person the benefit of the doubt.** 
It is always best to presume that someone is acting with the intentions of being clear and kind, and that they are unaware of how their words will affect others.
It's easy to not know something will cause offence and accidentally say the wrong thing.

The important thing is not the mistake, it is the learning and the change that comes from it.
So if someone says something that you feel is offensive, and you feel comfortable doing so, feel free to tell them.

> And if someone tells you you've said something that is offensive to them, believe them, and embrace the experience of learning more about that other culture, and extending your communication skills to not cause offense in future.

Reviewer 2: ```suggestion
And if someone tells you you've said something that is offensive to them, believe them, edit your message to remove or change the relevant text, and embrace the experience of learning more about that other culture, and extending your communication skills to not cause offense in future.
```
